ramls/examples/kbCredentials/kb_credentials_post_201_response.json

```diff
@@ -0,0 +1,18 @@
+{
+  "id": "2ffa1940-2cf6-48b1-8cc9-5e539c61d93f",
+  "type": "kbCredentials",
+  "attributes": {
+    "name": "University of Massachusetts",
+    "apiKey": "XXXX",
```

Comment:
Just want to confirm that the actual apiKey does not get returned in the POST response (we had **** it in the past)

Reply:
Update example with ****. Implementation will return the ***** API key.